ansible/configs/sap-integration/custom_workloads.yml

```diff
@@ -0,0 +1,109 @@
+- name: Set Ansible Python interpreter to k8s virtualenv
+  set_fact:
+    ansible_python_interpreter: /opt/virtualenvs/k8s/bin/python
+
+- name: Create openshift-storage namespace
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ns_storage.j2' ) | from_yaml }}"
+
+- name: Label nodes for OCS
+  # Using shell to support stream operations
+  shell: >
+    for n in $(oc get nodes | grep worker 
+    | awk {'print $1'}); do oc label node $n 
+    cluster.ocs.openshift.io/openshift-storage='';done
+
+- name: Create ocs operatorgroup
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ocs_operatorgroup.j2' ) | from_yaml }}"
+
+- name: Create ocs subscription
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/ocs_subscription.j2' ) | from_yaml }}"
+
+- name: Give the cluster some time to deploy both operators
+  pause:
+    minutes: 5
+
+- name: Create ocs cluster
+  k8s:
+    state: present
+    resource_definition: "{{ lookup( 'template', './files/k8s/storage_cluster.j2' ) | from_yaml }}"
+
+- name: Give the cluster some time so the storage classes are available
+  pause:
+    minutes: 5
+
+- name: Patch default Storage Class
+  shell: >
+    oc patch storageclass standard
+    -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+
+- name: Patch default Storage Class
+  shell: >
+    oc patch storageclass ocs-storagecluster-cephfs
```

Comment:
Can we use the k8s module here?

Reply:
I've had some issues in the past with k8s and patching objects doing weird stuff while merging information. Since the bastion will have the specific `oc` for the deployed OCP release I think this should always work as expected

Reply:
What issues? Has this given you the issue specifically here?